README.md

```diff
@@ -16,3 +16,12 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+
+To verify this claim I would time how long it takes to sort lists of increase size and take note of how the time it takes grows. If it grows faster than O(n) I would know that they are lying. However, technically they could sacrifice practical speed by making certain outputs take longer than others artifically. So that it grows linearly. If they do this for large enough sizes of n I might not be able to test high enough to prove them wrong.
```

Comment:
How would you be able to tell whether the measured runtimes are consistent with the $O(n)$ claim?